specs/simple-serialize.md

```diff
@@ -325,6 +326,7 @@ Instantiate a container with the full set of deserialized data, matching each me
 |:------------------------------------------|:----------------------------------------------------------------|
 | rawbytes has enough left for length       | ``len(rawbytes) > current_index + LENGTH_BYTES``                |
 | list is not greater than serialized bytes | ``len(rawbytes) > current_index + LENGTH_BYTES + total_length`` |
+| item index is at end of serialized bytes  | ``item_index == current_index + LENGTH_BYTES + total_length``   |
```

Comment:
This should probably checked for all types, not only for containers. So I'd not put it here, but at the beginning of the `Deserialize/Decode` section.